packages/calcite-components/src/components/slider/slider.scss

```diff
@@ -1,27 +1,54 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-slider-fill-color: Specifies the color of the fill.
+ * @prop --calcite-slider-graph-accent-color: Specifies the accent color of the graph.
```

Comment:
Should this use the `histogram` prop nomenclature? (Or, could `histogram` prop use `graph`, etc.,)